src/main/java/domain/pages/DirectoryPage.java

```diff
@@ -68,4 +73,21 @@ public char getPartyByRepresentative(String representativeName){
         }
         return allStatesArray;
     }
+
+    public List<WebElement> allRowInfoForSelectedState(String state){
+        List<WebElement> rows = driver.findElements(By.xpath(String.format(rowInfoXpath, state)));
+        return rows;
+    }
+
+    public Boolean checkFormatInTheRowsForSelectedState(String state){
+        Pattern pattern = Pattern.compile(regex);
+        Boolean result = false;
+        for (WebElement phone:allRowInfoForSelectedState("state-alabama")) {
```

Comment:
I assume thos method is common method for all states. But I also see alabama is hard coded... this is improper

Reply:
changed it to the same variable as the main method that it is included. I just wanted to give a try with Alabama state and forgot to correct.